cashctrl_ledger/ledger_utils.py

```diff
@@ -0,0 +1,20 @@
+"""
+This module provides utilities for handling and manipulating pandas DataFrames.
+"""
+
+import pandas as pd
+
+def df_to_str(target: pd.DataFrame) -> str:
+    """
+    Converts a DataFrame to a string, sorting columns and rows
+    for consistent comparison.
```

Comment:
Let's insist on the fact that we provide a unique representation regardless of ordering.
Also let's state what this function is useful for.
(Good practice to pass by ChatGPT to beautify the text):
```
Converts a DataFrame to a unique string representation, regardless of the column or row order.
This function is helpful to identify exact or close matches between collections of data frames.
```